tools/generate_resources_example.ps1

```diff
@@ -0,0 +1,28 @@
+Import-Module '.\src\CISDSCResourceGeneration\CISDSCResourceGeneration.psd1' -Force
+Remove-Item -Path '.\output' -Recurse -Force -ErrorAction 'SilentlyContinue'
+
```

Comment:
Should actual NetIDs be removed from these examples?

Reply:
You're right. I forget this is a public repo. Please do not approve until I make those changes.